Libraries/PeriphDrivers/Include/MAX32670/rtc.h

```diff
@@ -177,19 +177,35 @@ int MXC_RTC_GetFlags(void);
 int MXC_RTC_ClearFlags(int flags);
 
 /**
- * @brief     Get SubSecond
- * @retval    Returns subsecond value or E_BUSY, see /ref MXC_ERROR_CODES
+ * @brief     Get SubSecond or E_BUSY, see /ref MXC_ERROR_CODES
+ * @retval    Returns subsecond value
  */
-int MXC_RTC_GetSubSecond(void);
+int MXC_RTC_GetSubSecond(void) __attribute__((deprecated("Use MXC_RTC_GetSubSeconds() instead.")));
 
 /**
- * @brief     Get Second
- * @retval    returns second value or E_BUSY, see /ref MXC_ERROR_CODES
+ * @brief     This function stores the current value of the sub-seconds counter into a
+ *            pointer if the RTC is not busy. If the RTC is busy, an error is returned.
+ * @param     ssec   Pointer to the variable to store the current sub-seconds value.
+ * @retval    E_NO_ERROR if successful, otherwise an error code (see /ref MXC_ERROR_CODES).
  */
-int MXC_RTC_GetSecond(void);
+int MXC_RTC_GetSubSeconds(uint32_t *ssec);
 
 /**
- * @brief     Get the time using nuclear fusion. Or atomically. Something like that.
+ * @brief     Get Second or E_BUSY, see /ref MXC_ERROR_CODES
+ * @retval    returns second value
+ */
+int MXC_RTC_GetSecond(void) __attribute__((deprecated("Use MXC_RTC_GetSeconds() instead.")));
```

Comment:
Looks good, we just need to wrap the attributes in a check for GCC so we don't break IAR/Keil

```C
/**
 * @brief     Get Second
 * @retval    returns second value or E_BUSY, see /ref MXC_ERROR_CODES
 */
#ifdef __GNUC__
__attribute__((deprecated("Use MXC_RTC_GetSeconds() instead.")))
#endif
int MXC_RTC_GetSecond(void);
```